nodes/nodes/nodes-nodes-audit-log.adoc

```diff
@@ -0,0 +1,25 @@
+:context: nodes-nodes-working
+[id="nodes-nodes-audit-log-{context}"]
+= Viewing node audit logs
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+Audit provides a security-relevant chronological set of records documenting the
+sequence of activities that have affected system by individual users,
+administrators, or other components of the system.
+
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+
+include::modules/nodes-nodes-audit-log-basic.adoc[leveloffset=+1]
+
+include::modules/nodes-nodes-audit-log-basic-config.adoc[leveloffset=+1]
+
+include::modules/nodes-nodes-audit-log-basic-viewing.adoc[leveloffset=+1]
+
```

Comment:
All the three up urls are unaccessible

Reply:
These strings add the files to this document during the publishing process and are not accessible at this in this file directly.